pandas/_libs/tslibs/timestamps.pyi

```diff
@@ -131,8 +131,12 @@ class Timestamp(datetime):
     def astimezone(self, tz: _tzinfo | None) -> Self: ...  # type: ignore[override]
     def ctime(self) -> str: ...
     def isoformat(self, sep: str = ..., timespec: str = ...) -> str: ...
+    # Return type "datetime" of "strptime" incompatible with return type "Timestamp"
+    # in supertype "datetime"
     @classmethod
-    def strptime(cls, date_string: str, format: str) -> datetime: ...
+    def strptime(
+        cls, date_string: str, format: str
+    ) -> datetime: ...  # type: ignore[override]
```

Comment:
this ignore seems to be on the wrong line:

> pandas/_libs/tslibs/timestamps.pyi:137: error: Return type "datetime" of "strptime" incompatible with return type "Timestamp" in supertype "datetime"  [override]
> pandas/_libs/tslibs/timestamps.pyi:139: error: Unused "type: ignore" comment

Reply:
thx